packages/babel-plugin-emotion/src/index.js

```diff
@@ -97,6 +97,17 @@ export function replaceCssWithCallExpression(
 export function buildStyledCallExpression(identifier, tag, path, state, t) {
   const identifierName = getIdentifierName(path, t)
 
+  let stableClassName = `el-${hashString(state.file.opts.filename)}`
```

Comment:
I think filename + identifier name should be unique enough to generate a stable hash per emotion element instance.